.gitignore

```diff
@@ -15,3 +15,5 @@ bridge_generated.h
 #ignore keystores
 *.jks
 
+# Artefacts from running maker binary
+/data/maker/
```

Comment:
🙃
I love this comment: missing new line

Reply:
added :)